code/modules/jobs/job_types/enclave.dm

```diff
@@ -91,7 +91,7 @@
 /datum/job/enclave/enclavelt
 	title = "Enclave Lieutenant"
 	flag = F13USLT
-	total_positions = 1
+	total_positions = 0
```

Comment:
Don't do this.
God PLEASE don't do this.

Make the enclave template the "enclave-military" maptype, and put into an enclave folder.
You can then make your new enclave file.
This is for legacy and cleanliness sake.

If you need help doing this, Just ask me and I can do it tmrw.

Reply:
I don't understand what you're asking, so yes please help me tomorrow.

Reply:
Also this doesn't even remove the slots, just the readyup slots